test-graphql/client-test.js

```diff
@@ -0,0 +1,217 @@
+import assert from 'assert';
+import GraphQLJSClient from '@shopify/graphql-js-client';
+import Config from '../src-graphql/config';
+import Client from '../src-graphql/client';
+import types from '../types';
+import base64 from '../src-graphql/base64';
+import {singleProductFixture, multipleProductsFixture} from '../fixtures/product-fixture';
+import {fetchMock} from './fetch-mock-node';
+
+suite('client-test', () => {
+  const querySplitter = /[\s,]+/;
+
+  function tokens(queryString) {
+    return queryString.split(querySplitter).filter((token) => Boolean(token));
```

Comment:
@minasmart maybe we can expose this from `graphql-js-client` if we're going to be using it outside of it as well?